src/DI.php

```diff
@@ -443,11 +444,19 @@ public static function intro()
 	}
 
 	/**
-	 * @return Repository\PermissionSet
+	 * @return PermissionSet
 	 */
 	public static function permissionSet()
 	{
-		return self::$dice->create(Repository\PermissionSet::class);
+		return self::$dice->create(PermissionSet::class);
+	}
+
+	/**
+	 * @return \Friendica\Security\PermissionSet\Factory\PermissionSet
+	 */
+	public static function permissionSetFactory()
+	{
+		return self::$dice->create(\Friendica\Security\PermissionSet\Factory\PermissionSet::class);
 	}
```

Comment:
Please move the return type declaration to the language syntax now that we can:
```suggestion
	public static function permissionSetFactory(): \Friendica\Security\PermissionSet\Factory\PermissionSet
	{
		return self::$dice->create(\Friendica\Security\PermissionSet\Factory\PermissionSet::class);
	}
```